README.md

```diff
@@ -105,10 +108,13 @@ func main() {
 	// Produce messages to topic (asynchronously)
 	topic := "myTopic"
 	for _, word := range []string{"Welcome", "to", "the", "Confluent", "Kafka", "Golang", "client"} {
-		p.Produce(&kafka.Message{
+		err = p.Produce(&kafka.Message{
 			TopicPartition: kafka.TopicPartition{Topic: &topic, Partition: kafka.PartitionAny},
 			Value:          []byte(word),
 		}, nil)
+                if err != nil {
+                    fmt.Printf("Message could not be enqueued: %v\n", err)
```

Comment:
Would `fmt.Println("Message could not be enqueued: ", err)` be more readable and concise?

Reply:
All other impressions are using `fmt.Printf()`, I kept the same.

Reply:
Yes, because they need to interpolate values in the middle of their strings... here you are only appending a value, so you don't need `printf` and friends.

Reply:
Please use fmt.Printf() since it is consistent with other code, and more explicit what formatting is desired.

Reply:
Yes, make sence. I fixed it.

Reply:
This is also using whitespaces rather than tabs.